### ISSUE TITLE
How to set weight for each class?

### ISSUE THREAD

**Issue by heroxrq** [Microsoft/LightGBM#434]
I faced a problem to train a 6 classes classifier. Since data is unbalanced and I want to modify the loss function. Can lightgbm set weight of each class like xgboost?

**Comment by Laurae2** [comment]
Use the weight parameter: https://github.com/Microsoft/LightGBM/blob/master/docs/Parameters.md#weight-data

**Comment by unnir** [comment]
the link is broken

**Comment by guolinke** [comment]
@unnir https://github.com/Microsoft/LightGBM/blob/master/docs/Parameters.rst#weight-data
